src/scss/OutSystemsUI.scss

```diff
@@ -403,7 +403,7 @@ Functions & Mixins
 /*! 6.3.18. TimePicker */
 @import '../scripts/OSFramework/OSUI/Pattern/TimePicker/scss/timepicker';
 /*! 6.3.19. Video */
-@import '04-patterns/03-interaction/video';
```

Comment:
@bmarcelino-fe I think with what you did the deprecated-video scss will no longer be compiled to the OutSystemsUI.css

Reply:
Solved the issue on this. Now the video deprecated SCSS is compiled. Missed that we need to changed that on PatternsDeprecated.js